pos_second_uom/models/product.py

```diff
@@ -0,0 +1,11 @@
+from odoo import fields, models
+
+
+class PropertyPlan(models.Model):
```

Comment:
```suggestion
class ProductTemplate(models.Model):
```
If you're copying and pasting the code, please take a moment to double-check everything.